3-async-await/solution.js

```diff
@@ -20,16 +20,43 @@ Example:
 9. as extra challenge: add Promise.race() and Promise.any(), and try to get the idea of what happens
 */
 
-function solution() {
+const productMethod = require('./products')
+const priceMethod = require('./prices')
+
+async function solution() {
     // YOUR SOLUTION GOES HERE
 
     // You generate your id value here
-
+    const id = parseInt(Date.now().toString().slice(-2))
+
+    const promise1 = new Promise((resolve) => {
+        resolve(productMethod(id))
+    })
+    const promise2 = new Promise((resolve) => {
```

Comment:
Use a better naming..
```suggestion
    const pricePromise = new Promise((resolve) => {
```